addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -443,11 +444,27 @@ class MessageList extends Component {
             return false;
         }
         const loadMoreRect = loadMore.getBoundingClientRect();
-        const elRect = this.el.getBoundingClientRect();
+        const elRect = this.scrollableParent.getBoundingClientRect();
         const isInvisible = loadMoreRect.top > elRect.bottom || loadMoreRect.bottom < elRect.top;
         return !isInvisible;
     }
 
+    /**
+     * @private
+     * @param {Element}
+     * @returns {Element} closest scrollable node element
+     */
+    getScrollableParent(node) {
+        if (node == null) {
+            return null;
+        }
+        if (node.scrollHeight > node.clientHeight) {
+            return node;
+        } else {
```

Comment:
This is not the correct code to know if something is scrollable, it just tells if there is an actual scrollbar currently. If the message list is small, this condition will make it go up to any random Element that we don't want to target.

So this shouldn't be generic, while we are at it making crappy hack, I guess we might as well just directly use either `.closest('.o_Chatter_scrollPanel')` if it is found, and otherwise return current element.
Then the method must probably be renamed to something else to match what it does.

Reply:
@seb-odoo To me using selector which is not part of our current element looks crappy, ```o_Chatter_scrollPanel``` is parent of the current element so fetching it specifically looks odd to me, that's why I created ```getSCrollableParent``` method which is something like jQuery UI getScrollableParent and dynamic in manner.